src/widgets/kernel/qtooltip.cpp

```diff
@@ -181,6 +209,23 @@ void QTipLabel::reuseTip(const QString &text, int msecDisplayTime, const QPoint
     restartExpireTimer(msecDisplayTime);
 }
 
+void QTipLabel::reuseTip(QLayout* layout, int msecDisplayTime, const QPoint& pos)
+{
+#ifndef QT_NO_STYLE_STYLESHEET
+    if (styleSheetParent) {
+        disconnect(styleSheetParent, SIGNAL(destroyed()),
+            QTipLabel::instance, SLOT(styleSheetParentDestroyed()));
+        styleSheetParent = nullptr;
+    }
+#endif
+
+    setText("");
```

Comment:
Could possibly use `https://doc.qt.io/qt-6/qlabel.html#clear` here? Not sure what the difference would be if any

Reply:
I'm not sure of any advantages it will give us, but I'll try to use it